document/src/vespa/document/datatype/documenttype.h

```diff
@@ -34,6 +34,8 @@ class DocumentType : public StructuredDataType {
 
         FieldSet(const FieldSet&) = default;
         FieldSet(FieldSet&&) noexcept = default;
+        FieldSet& operator=(const FieldSet&) = default;
+        FieldSet& operator=(FieldSet&&) = default;
```

Comment:
You should add noexcept, both here and on the move constructor.

Reply:
Done.